logrus_test.go

```diff
@@ -372,6 +373,21 @@ func TestParseLevel(t *testing.T) {
 	assert.Equal(t, "not a valid logrus Level: \"invalid\"", err.Error())
 }
 
+func TestUnmarshalText(t *testing.T) {
+	u, _ := (interface{}(&(AllLevels[0]))).(encoding.TextUnmarshaler)
```

Comment:
You can just declare a Level variable here

Reply:
Would you tell me what the “Level variable” means? Suggesting to stop using `AllLevels`?

Reply:
Just something like:
```
var u Level
```